pkgs/pkgs-lib/tests/formats.nix

```diff
@@ -425,4 +425,48 @@ in runBuildTests {
       \u0627\u0644\u062c\u0628\u0631 = \u0623\u0643\u062b\u0631 \u0645\u0646 \u0645\u062c\u0631\u062f \u0623\u0631\u0642\u0627\u0645
     '';
   };
+
+  phpAtoms = shouldPass rec {
+    format = formats.php { finalVariable = "$config"; };
+    input = {
+      null = null;
+      false = false;
+      true = true;
+      int = 10;
+      float = 3.141;
+      str = "foo";
+      str_special = "foo\ntesthello'''";
+      attrs.foo = null;
```

Comment:
If adding `attrs.bar = "bar";` just under this line, will it convert it correctly into a single `attrs => [ 'bar' => 'bar', 'foo' => null ]` ?

Reply:
Yeah. From nix perspective `attrs.bar = "bar"; attrs.foo = null;` is equal to `attrs = { bar = "bar"; foo = null; };`.

Reply:
That's cool then.